lib/Serialization/ModuleFile.cpp

```diff
@@ -1996,16 +2230,36 @@ ModuleFile::loadNamedMembers(const IterableDeclContext *IDC, DeclBaseName N,
     DeclMembersTables[subTableOffset];
   if (!subTable) {
     BCOffsetRAII restoreOffset(DeclMemberTablesCursor);
-    DeclMemberTablesCursor.JumpToBit(subTableOffset);
-    auto entry = DeclMemberTablesCursor.advance();
+    if (llvm::Error Err = DeclMemberTablesCursor.JumpToBit(subTableOffset)) {
+      // FIXME this drops the error on the floor.
+      consumeError(std::move(Err));
+      error();
```

Comment:
From here through pretty much the rest of the file, it's too late to recover, so you can use `fatal(std::move(Err))` instead.

Reply:
OK I removed the FIXME.

Reply:
I updated this and subsequent `consumeError`.